armory/data/adversarial_datasets.py

```diff
@@ -138,7 +150,7 @@ def resisc45_adversarial_224x224(
         variable_length=False,
         cache_dataset=cache_dataset,
         framework=framework,
-        lambda_map=lambda x, y: ((x[clean_key], x[adversarial_key]), y),
+        lambda_map=lambda_map,
     )
```

Comment:
```supervised_xy_keys=("images", "label"),``` should be changed to ```supervised_xy_keys=("images", "labels"),```

Reply:
Thanks, David. Still getting the error: 
`tensorflow.python.framework.errors_impl.InvalidArgumentError: {{function_node __inference_Dataset_map_ExampleParser.parse_example_20107}} Feature: labels/adversarial_univpatch (data type: int64) is required but could not be found.
`

Reply:
Please set [cache_dataset](https://github.com/twosixlabs/armory/blob/b8b9c35c94ac2b47fb07100de3fceaca003b1213/armory/data/adversarial_datasets.py#L110) to be false. Right now the workflow requires us to update that manually when a dataset is changed. If there was a different dataset with 1.0.1 that you ran locally, you may need to clear out the related directories in your datasets directory as well. It may actually make sense to increment the version if the structure of the dataset is changing.

Reply:
Thanks -- still same error. It's not a different dataset; just an update to the dataset builder file.

Reply:
True, but that causes the processed dataset to be different - so you'll probably need to increment the version number.

Reply:
That worked! Thanks for picking up on that and explaining it!